Sources/PackageDescription/PackageDescription.swift

```diff
@@ -395,6 +398,16 @@ public enum SystemPackageProvider {
     public static func brew(_ packages: [String]) -> SystemPackageProvider {
         return .brewItem(packages)
     }
+    
+    /// Creates a system package provider with a list of installable packages
+    /// for people who use the MacPorts package manager on macOS.
+    ///
+    /// - Parameter packages: The list of package names.
+    ///
+    /// - Returns: A package provider.
+    public static func port(_ packages: [String]) -> SystemPackageProvider {
```

Comment:
Let's make the name more obvious, applicable to other places where `port` was used as well.
```suggestion
    public static func macPorts(_ packages: [String]) -> SystemPackageProvider {
```

Reply:
1. We should consider the consistency of API.
For `HomeBrew` package manager (`brew install xx`), we use `brew` here.
For `apt-get` package manager (`apt install xx`), we use `apt` here.

IMO for `MacPorts` package manager (`port install xx`), we should use `port` here.

2. Swift have a namespace system.
So I'm not expecting it to appear in the wrong place. 
We can only use `.port(["xx"])` if the callee accepts a `SystemPackageProvider` params.

Reply:
Sticking to the name of a CLI command, e.g. `brew` or `apt` there was a mistake. There are enough package managers CLI named `port` or `pkg` on BSD systems for example. By squatting `port` here we'll create a mess for possible future BSD support. I'm not sure how exactly limited means of namespacing in Swift would help.

Reply:
Additionally, using plain `port` creates confusion for a reader. I'm skimming through a package and see `.port` there without any disambiguation. Is that a Mac port, FreeBSD port, or OpenBSD port? Brevity and consistency is good, but not at the cost of reduced readability and increased chance for confusion.

Reply:
> Sticking to the name of a CLI command, e.g. `brew` or `apt` there was a mistake. There are enough package manager CLI commands named `port` or `pkg` on BSD systems for example. By squatting `port` here we'll create a mess for possible future BSD support. I'm not sure how exactly limited means of namespacing in Swift would help.

Got it. I'll update the feedback to the forum post.

Reply:
> Sticking to the name of a CLI command, e.g. `brew` or `apt` there was a mistake. There are enough package manager CLI commands named `port` or `pkg` on BSD systems for example. By squatting `port` here we'll create a mess for possible future BSD support. I'm not sure how exactly limited means of namespacing in Swift would help.

I'm convinced by the logic here and happy to make such change. but I'd like to know how other reviewers and the community thinking of this before making a new commit.

Reply:
@tomerd WDYT? Does deprecating old names and introducing a new one require an SE pitch, proposal, and review? Or should we just introduce a new `macPorts` here without an SE process, and discuss deprecation of the old ones separately?

Reply:
Any progress on this? Can we merge it with `@available(_PackageDescription, introduced: 999.0)` the same as #5486 ?

Reply:
@Kyle-Ye I prefer the `.macPorts(` since MacPort is specific to macOS and is not 100% equal to the `port` in other systems (e.g. FreeBSD ports, etc.).

Reply:
@MaxDesiatov Please do consider respond to Kyle-Ye's response above. This PR solves the issue that in Mainland China one can hardly use Brew without taking legal risks to use a cross-GFW virtual private network. MacPorts is accessible since its mirrors are available inside the Mainland China.

(Also, again I suggest treating `MacPorts` and non-mac `Ports` differently with individual APIs since the availability of a package might differ between them.)